### PR TITLE
Fix key set paginator

### PR DESCRIPTION
**Pull request by romkatsu**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #43 

I don't really like this decision. There was also an option to accept callbale.
What do you think about this decision?

### CODE REVIEW

File: src/Paginator/KeysetPaginator.php [hunk 1]
```diff
@@ -296,4 +297,19 @@ private function previousPageExist(ReadableDataInterface $dataReader, Sort $sort
         }
         return false;
     }
+
+    private function getValueFromItem($item, string $field)
+    {
+        $getterName = $this->getGetterName($field);
+        if (is_object($item) && method_exists($item, $getterName)) {
```
Comment (roxblnfk) [inline src/Paginator/KeysetPaginator.php:304]:
```suggestion
        if (is_object($item) && is_callable([$item, $getterName])) {
```
https://3v4l.org/JGgoM

File: src/Paginator/KeysetPaginator.php [hunk 2]
```diff
@@ -296,4 +297,19 @@ private function previousPageExist(ReadableDataInterface $dataReader, Sort $sort
...
+            return $item->$getterName();
+        }
+
+        return ArrayHelper::getValue($item, $field);
+    }
+
+    private function getGetterName(string $field): string
```
Comment (roxblnfk) [inline src/Paginator/KeysetPaginator.php:311]: I think that there is no need to allocate a separate method for this

**Review by roxblnfk — CHANGES_REQUESTED** [review]

File: src/Paginator/KeysetPaginator.php (hunk 1 above)
Comment (romkatsu) [reply src/Paginator/KeysetPaginator.php]: fixed

File: src/Paginator/KeysetPaginator.php (hunk 2 above)
Comment (romkatsu) [reply src/Paginator/KeysetPaginator.php]: remove

**Review by samdark — APPROVED** [review]

**Review by roxblnfk — APPROVED** [review]

**Review by hiqsol — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**